### DISCUSSION TITLE
No level 1 title

### DISCUSSION THREAD

**Discussion by Loveyless** [vuejs/vitepress#806] [General]
Is this level 1 heading intentionally left out or am I mistaken？

![2c5ac6beae0c2de05f0014795bfaa71](https://user-images.githubusercontent.com/88988179/174375333-34f298d8-2736-43f7-985a-cdcac8ae7115.png)
![80acad0d970ab439e5c1b5aa1bbc228](https://user-images.githubusercontent.com/88988179/174375475-66d2e068-24cd-4fef-954e-70d71b18ad36.png)

No level 1 title
But I went to the Vuejs website

![b975cacee8dd21db7869adde29bc3c8](https://user-images.githubusercontent.com/88988179/174377372-15522adb-3f44-4461-88f6-759fad835e95.png)

So I had to split my code
But a lot of times I don't want to do that

**Comment by brc-dd** [comment]
It is intentional. We are discouraging multiple `h1` elements per page. In Vue.js docs we have nested outline (`h2` and `h3` titles). To get that you need to put `outline: deep` in your frontmatter.

**Reply by Loveyless** [reply]
I understand that will make people look more intuitive. thank you！
